diff --git a/node_modules/@firebase/util/dist/node-esm/package.json b/node_modules/@firebase/util/dist/node-esm/package.json
deleted file mode 100644
index 7c34deb..0000000
--- a/node_modules/@firebase/util/dist/node-esm/package.json
+++ /dev/null
@@ -1 +0,0 @@
-{"type":"module"}
\ No newline at end of file
diff --git a/node_modules/@firebase/util/dist/postinstall.mjs b/node_modules/@firebase/util/dist/postinstall.mjs
index 0c02b08..ec2d55b 100644
--- a/node_modules/@firebase/util/dist/postinstall.mjs
+++ b/node_modules/@firebase/util/dist/postinstall.mjs
@@ -1,2 +1 @@
-const getDefaultsFromPostinstall = () => (undefined);
-export { getDefaultsFromPostinstall };
\ No newline at end of file
+export function getDefaultsFromPostinstall() { return undefined; }
diff --git a/node_modules/@firebase/util/dist/test/base64.test.d.ts b/node_modules/@firebase/util/dist/test/base64.test.d.ts
deleted file mode 100644
index cb0ff5c..0000000
--- a/node_modules/@firebase/util/dist/test/base64.test.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export {};
diff --git a/node_modules/@firebase/util/dist/test/deepCopy.test.d.ts b/node_modules/@firebase/util/dist/test/deepCopy.test.d.ts
deleted file mode 100644
index cb0ff5c..0000000
--- a/node_modules/@firebase/util/dist/test/deepCopy.test.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export {};
diff --git a/node_modules/@firebase/util/dist/test/defaults.test.d.ts b/node_modules/@firebase/util/dist/test/defaults.test.d.ts
deleted file mode 100644
index cb0ff5c..0000000
--- a/node_modules/@firebase/util/dist/test/defaults.test.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export {};
diff --git a/node_modules/@firebase/util/dist/test/emulator.test.d.ts b/node_modules/@firebase/util/dist/test/emulator.test.d.ts
deleted file mode 100644
index cb0ff5c..0000000
--- a/node_modules/@firebase/util/dist/test/emulator.test.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export {};
diff --git a/node_modules/@firebase/util/dist/test/errors.test.d.ts b/node_modules/@firebase/util/dist/test/errors.test.d.ts
deleted file mode 100644
index cb0ff5c..0000000
--- a/node_modules/@firebase/util/dist/test/errors.test.d.ts
+++ /dev/null
@@ -1 +0,0 @@
-export {};
